internal/testdata/testdatautils.go

```diff
@@ -0,0 +1,21 @@
+package testdata
+
+import "clouditor.io/clouditor/api/auth"
+
+func NewUser1() *auth.User {
```

Comment:
The api/user stuff is legacy and we should really remove this at some point (maybe now!)

Reply:
I think that is a task for you :wink:

Reply:
Done. can you double-check if that is ok? Otherwise we can merge it.